recipe/meta.yaml

```diff
@@ -1,58 +1,70 @@
-{% set version = "1.1.7" %}
+{% set name = "poetry" %}
+{% set version = "1.4.0" %}
 
 package:
-  name: poetry
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/poetry/poetry-{{ version }}.tar.gz
-  sha256: 3833c7b22411b8393e8e594fede94f0ef7a5260c19f51e272cce76512cffe6c8
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/poetry-{{ version }}.tar.gz
+  sha256: 151ad741e163a329c8b13ea602dde979b7616fc350cfcff74b604e93263934a8
 
 build:
-  number: 1
-  skip: true  # [py<35]
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install --no-deps . -vv
   entry_points:
-    - poetry = poetry.console:main
+    - poetry = poetry.console.application:main
 
 requirements:
   host:
     - python
     - pip
-    - poetry-core >=1.0.0,<2.0.0
+    - setuptools
+    - wheel
+    - poetry-core 1.5.1
   run:
     - python
-    - cachecontrol >=0.12.4,<0.13.0
-    # cachecontrol:filecache
-    - lockfile >=0.9
-    - cachy >=0.3.0,<0.4.0
-    - cleo >=0.8.1,<0.9.0
-    - clikit >=0.6.2,<0.7.0
-    - crashtest >=0.3.0,<0.4.0  # [py>=36]
+    - poetry-core 1.5.1
+    - poetry-plugin-export >=1.3.0,<2.0.0
+    - python-build >=0.10.0,<0.11.0
+    - cachecontrol >=0.12.9,<0.13.0
+
+    # cachecontrol filecache extra
```

Comment:
Move `# cachecontrol filecache extra` to `run_constrsained`

Reply:
Not sure if your comment above is the same as this?